src/js/view/controls/components/settings/menu.js

```diff
@@ -19,22 +19,51 @@ export function SettingsMenu(onVisibility, onSubmenuAdded, onMenuEmpty) {
 
     const settingsMenuElement = createElement(SettingsMenuTemplate());
 
-    const closeOnEnter = function(evt) {
-        if (evt && evt.keyCode === 27) {
-            instance.close(evt);
+    const handleKeyDown = function(evt) {
+        if (evt) {
```

Comment:
As long as this function is only used as an event handler, you can count on `evt` being defined and do not need this `if` statement.